samples/bot-task-module/nodejs/pages/TaskModuleTab.js

```diff
@@ -1,15 +1,15 @@
 //- Copyright (c) Microsoft Corporation.
 //- All rights reserved.
-
 // microsoftTeams app Initilize
-microsoftTeams.app.initialize().then(() => {
+ microsoftTeams.app.initialize().then(() => {
     var taskModuleButtons = document.getElementsByClassName("taskModuleButton");
     if (taskModuleButtons.length > 0) {
+        fetch(`${window.location.origin}/AuthTab`).then(response => response.json()).then(data => {
 
         //config your appid and baseurl
         var config = {
-            MicrosoftAppID: "<<Microsoft-App-ID>>",
-            BaseUrl: "<<BaseUrl>>"
+            MicrosoftAppID: data.MicrosoftAppId,
+            BaseUrl: `${window.location.origin}`
```

Comment:
Will it work in Desktop and mobile clients?  ${window.location.origin}

Reply:
@Harikrishnan-MSFT - Please run build pipeline as well.

Reply:
It is working in Desktop clients.

Reply:
****Build** Pipeline -** 
![image](https://github.com/OfficeDev/Microsoft-Teams-Samples/assets/117716554/6d12ca7e-4209-4a80-9c44-42f38b8c21cf)

https://github.com/OfficeDev/Microsoft-Teams-Samples/actions/runs/5423570458